collection-scripts/gather_network_logs

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+BASE_COLLECTION_PATH="must-gather"
+NETWORK_LOG_PATH="${BASE_COLLECTION_PATH}/network_logs/"
+
+mkdir -p ${NETWORK_LOG_PATH}/
+
+WORKERS=$(oc get nodes --no-headers -o custom-columns=":metadata.name")
```

Comment:
This is not restricted to workers (consider re-naming this to nodes). 
Or ad a label selector like you do below (for Masters)

Reply:
agreed, this should be on all nodes.